.github/workflows/test.yml

```diff
@@ -150,9 +150,11 @@ jobs:
         run: |
           make coverage-check
 
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3
+      - name: Publish coverage report to Code Climate
```

Comment:
So strictly speaking we are replacing "codecov" with "codeclimate" providing feedback on technical debt analysis in addition to testing coverage.

Reply:
For the time being Yes. In the future, we will try to configure go format and go linting using codeclimate itself.